esp/src/src/nls/es/hpcc.ts

```diff
@@ -785,6 +825,9 @@ export = {
     ThorNetworkAddress: "Dirección de red de Thor",
     ThorProcess: "Proceso de Thor",
     Time: "Tiempo",
+    TimeMaxTotalExecuteMinutes: "Tiempo máximo total de minutos de ejecución",
+    TimeMeanTotalExecuteMinutes: "Tiempo promedio total de minutos de ejecución",
+    TimeMinTotalExecuteMinutes: "Tiempo mínimo Total Ejecutar Minutos",
```

Comment:
"Total" capitalized here, but not in previous 2 messages, same for Ejecutar/ejecucion and minutos.

In fact these 3 messages should be:
Maximo/Total/Minomo tiempo total de ejecucion en minutos.